app/styles/global.css

```diff
@@ -1,3 +1,29 @@
 /* This is a global CSS file  */
 @import url('react-big-calendar/lib/css/react-big-calendar.css');
 @import url('react-big-calendar/lib/addons/dragAndDrop/styles.css');
+
+/* overriding react-big-calendar's css properties */
+.rbc-event {
+  background-color: #e7e5e4;
```

Comment:
Is this through the pallete that was softly agreed upon in the discussion.
Was it discussed with @yesyash

Reply:
@harshith-venkatesh the pallete is still pending. For now I have used the hex colors from the Figma file. Later when we have the pallet, we can replace the colors from the whole codebase.

Reply:
Thanks for the update @iRohitGaur